.circleci/config.yml

```diff
@@ -0,0 +1,160 @@
+version: 2.1
+
+checkout:
+  post:
+    - >
+      if [ -n "$CI_PULL_REQUEST" ]; then
+        PR_ID=${CI_PULL_REQUEST##*/}
+        git fetch origin +refs/pull/$PR_ID/merge:
+        git checkout -qf FETCH_HEAD
+      fi
+
+
+executors:
+  py27:
+    docker:
+      - image: circleci/python:2.7
+
+  py36:
+    docker:
+      - image: circleci/python:3.6
+
+  wagon_generator:
+    machine:
+      image: ubuntu-1604:201903-01
+
+
+commands:
+  run_unittest_py27:
```

Comment:
unitests should run on py36 too?

Reply:
There aren't any at the moment I think, sorry- forgot to remove the command as well as the job.

Reply:
(the tests are openstack specific at the moment)

Reply:
(removed this)